arma3/userconfig/task_force_radio/radio_settings.hpp

```diff
@@ -1,19 +0,0 @@
-﻿
-// =================================================
-// Server side only
-// =================================================
-tf_no_auto_long_range_radio = 0;
-TF_give_personal_radio_to_regular_soldier = 0;
-TF_give_microdagr_to_soldier = 1;
-tf_same_sw_frequencies_for_side = 0;
-tf_same_lr_frequencies_for_side = 0;
-tf_same_dd_frequencies_for_side = 0;
-// =================================================
-// END: Server side only
-// =================================================
-// Client side
-// =================================================
-tf_default_radioVolume = 7;
```

Comment:
I dont know if that featured is used by a susceptible amount of people, for me, could be removed with any problem.

The other option is making a dialog.